stan/math/prim/mat/fun/multiply.hpp

```diff
@@ -4,6 +4,9 @@
 #include <stan/math/prim/mat/fun/Eigen.hpp>
 #include <stan/math/prim/arr/err/check_matching_sizes.hpp>
 #include <stan/math/prim/mat/err/check_multiplicable.hpp>
+#ifdef STAN_OPENCL
+#include <stan/math/opencl/multiply.hpp>
```

Comment:
Do you think it would be better to have `prim` and `rev` just include from opencl.hpp? Kind of like how we are trying to zip up the rest of stan it may be good to just have one include for everything

Reply:
Oh yes, absolutely, great catch. Everything outside of `/opencl` should only include that. Will also extend the `test-math-dependencies`.